functions/src/src/AddEvent.ts

```diff
@@ -23,22 +21,11 @@ import { Event } from '../models/Event'
  */
 
 export async function addEvent(name: string, details: string, date: Date, location: string,
-                                points: number, point_type_id: number, point_type_name: string,
-                                point_type_description: string, house: string, creator_id: string): Promise<Boolean> {
-    let event = new Event(name, details, date, location, points, point_type_id, point_type_name,
-                        point_type_description, house, creator_id, "")
+                                points: number, point_type: PointType, house: string, creator_id: string): Promise<void> {
+    let event = new Event(name, details, date, location, points, Number(point_type.id), point_type.name.toString(),
```

Comment:
In typescript, like in Java, several basic types have wrapper classes that have the same name but capitalized. (String, Number, Boolean, etc). The format that the other models follow is to use the basic type as the types for the fields so that we don't have to wrap them like Number(point_type.id). Can you have Event use the basic types?

Reply:
Event does have the basic types. This is just the only way I knew to convert from string to number since Firebase sees document IDs as strings